working/1426-extension-types/feature-specification-views.md

```diff
@@ -718,23 +743,26 @@ A compile-time error occurs if `V1` is a type name or a parameterized type
 which occurs as a superview in a view declaration _DV_, but `V1` does not
 denote a view type.
 
+A compile-time error occurs if `V1` is a superview of `V1`, directly or
```

Comment:
Since `V1` is a view *type*, not a view *declaration*, I think this permits stuff like this:

```dart
view class V<T> implements V<List<T>> {}
```

(because `V<T>`, `V<List<T>>`, `V<List<List<T>>>`, and so on are all distinct types).

I don't think we want to permit that.  Could we say something like this instead?

> A compile-time error occurs if any superview of _DV_ is the type `View` or `View<...>`. *As usual, subtype cycles are not allowed.*